store/store.go

```diff
@@ -885,7 +885,7 @@ func (s *Store) fakeChannels(snapID string, user *auth.UserState) (map[string]*s
 
 	var results struct {
 		Payload struct {
-			Refs []*snap.Ref `json:"clickindex:package"`
+			SnapInfos []*snapDetails `json:"clickindex:package"`
```

Comment:
s/SnapInfos/SnapDetails/?  Otherwise the loop below misleads one into thinking the values are infos.